JS/Lesson35/savas/my-app/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link, useMatch, useResolvedPath } from "react-router-dom";
+import "./navbar.css";
+const CustomLink = ({ to, pageName }) => {
+  const resolved = useResolvedPath(to);
+  const isActive = useMatch({ path: resolved.pathname, end: true });
+  return (
+    <li className={`item ${isActive ? "active" : ""}`}>
+      <Link to={to}>{pageName}</Link>
```

Comment:
There's a NavLink element that does what you need here :) 
https://reactrouter.com/en/main/components/nav-link

As long as you use BrowserRouter (see index.js file in the main directory), the NavLink element should be available for you.

Reply:
I changed it, but I don't know that I did it right :)

Reply:
Could it be that you did not push your changes? I cannot see a change